src/idl/protos.d.ts

```diff
@@ -0,0 +1,248 @@
+import * as $protobuf from 'protobufjs'
```

Comment:
This file, `protos.d.ts`, is compiled from `.proto` definitions. You should disregard it.

Reply:
Should this be in git then? Normally you tend to avoid putting compiled stuff in to prevent polluting git with huge files. Instead we could put a "yarn compile" step in "yarn start"...?

Reply:
It's easier for development when it's available compiled right away, but now that you pointed this out, I realized this can be compiled in `yarn prepare`, so it should actually be seamless for the developer, without committing the file. 🎉